src/suricata.c

```diff
@@ -1396,6 +1398,9 @@ static TmEcode ParseCommandLine(int argc, char** argv, SCInstance *suri)
         {"simulate-packet-tcp-ssn-memcap", required_argument, 0, 0},
         {"simulate-packet-defrag-memcap", required_argument, 0, 0},
         {"simulate-alert-queue-realloc-failure", 0, 0, 0},
+
+        {"simulate-skip-prefilter", 0, &g_skip_prefilter, 1 },
```

Comment:
Is this a simulation, or actually skipping the pre-filter?

Reply:
not a simulation. Perhaps it should use a `qa` prefix or something. It needs to be clear it's not intended to be used by actual users.